app/components/Calendar.client.tsx

```diff
@@ -0,0 +1,11 @@
+import TUICalendar from '@toast-ui/react-calendar';
+
+function Calendar() {
+  return (
+    <div>
+      <TUICalendar height="100vh" />
+    </div>
+  );
+}
```

Comment:
NIT: Arrow function and can we get rid of `return` keyword and directly use `()`?